code/graphics/generic.cpp

```diff
@@ -217,7 +217,7 @@ int generic_anim_stream(generic_anim *ga, const bool cache)
 				ga->png.anim = new apng::apng_ani(ga->filename, cache);
 			}
 			catch (const apng::ApngException& e) {
-				mprintf(("Failed to load apng: %s\n", e.what() ));
+				nprintf(("apng","Failed to load apng: %s\n", e.what() ));
```

Comment:
This should be displayed

Reply:
I have a minor vendeta against these apng messages... in my most recent level load in WM I got almost 200 lines of this complaining about *glow maps* that are all valid. This was supposedly fixed after I filed it for #3644, it's the exact smage message, but I can't remember if I verified the fix at the time or if this was just working as intended.

I can restore them I guess, if they're ever actually useful for casual debugging.